### PR TITLE
[BUGFIX] Minor bug fixes while reading config.properties file

### PR DESCRIPTION
**Pull request by Om-Gujarathi**
## Description
Fixes - #6 and #7 

## Changes Made

1. Used the `with` statement to ensure the file is closed properly after reading.
2. Used a `try` block to handle `FileNotFoundError` and other exceptions gracefully.
3. Added a condition `if prop.strip():` to skip empty lines.

## Definition of Done

Before submitting this pull request, please ensure that the following criteria have been met:

- [x] All automated tests have passed successfully.
- [x] All manual tests have passed successfully.
- [x] Code has been reviewed by at least one other team member.
- [x] Code has been properly documented and commented as needed.
- [x] All new and existing code adheres to our project's coding standards.
- [x] All dependencies have been added or removed from the project's README or other documentation as needed.
- [x] Any relevant documentation or help files have been updated to reflect the changes made in this pull request.
- [x] Any necessary database migrations have been run.
- [x] Any relevant UI changes have been reviewed and approved by the UI/UX team.

### CODE REVIEW

File: ingen/utils/properties.py [hunk 1]
```diff
@@ -30,12 +30,23 @@ def initialize_properties(self):
...
-                for prop in properties_line:
-                    list_property = prop.split("=")
-                    self.property_map[list_property[0]] = list_property[1]
+            try:
+                if path is not None:
+                    with open(os.path.join(path, "config.properties"), 'r') as file:
+                        properties_line = file.read().splitlines()
+                        for prop in properties_line:
+                            if prop.strip():
+                                print(prop)
```
Comment (shpiyu) [inline ingen/utils/properties.py:39]: you may remove this `print` statement

**Review by shpiyu — CHANGES_REQUESTED** [review]
Thanks for your contribution Om, I've run the run code locally and it fixes both the issues. Thanks a lot. Could you please remove the print statement and then we can merge the PR.

**Review by shpiyu — CHANGES_REQUESTED** [review]
Thank you!

**Review by shpiyu — APPROVED** [review]

**Review by Jatin-8898 — APPROVED** [review]

**Review by swarnadhakad — COMMENTED** [review]
LGTM

**Pull request merged**